docs/architecture/pupil-check/readme.md

```diff
@@ -1,9 +1,17 @@
-# Check Submission Payload
+# Pupil Check
 
-An example payload can be found [here](../check-submission/check-payload.json)
+- [Back to Architecture Home](../readme.md)
+
+## Overview
+
+The Pupil Check is a browser based Angular application written in Typescript that is used by pupils to complete the Multiplication Tables Check.  The application is designed to be accessible to all pupils, including those with special educational needs and disabilities.  The application is designed to be used on a variety of devices, including desktops, laptops, tablets and mobile phones.
```

Comment:
It is not entirely factually correct to say it is designed to work on mobile phones.  Although it does work on mobile phones, it is not designed to be taken on them.  Tablets are realistically the smallest form factor we expect to be used.